conda_devenv/devenv.py

```diff
@@ -12,12 +12,37 @@ def render_jinja(contents, filename, is_included):
     import sys
     import platform
 
+    pyversion = platform.python_version().split('.')
+
+    iswin = sys.platform.startswith('win')
+    islinux = sys.platform.startswith('linux')
+    isosx = sys.platform.startswith('darwin')
+
+    is32bit = '32bit' == platform.architecture()[0]
+    is64bit = not is32bit
+
     jinja_dict = {
         "is_included": is_included,
         "os": os,
         "platform": platform,
         "root": os.path.dirname(os.path.abspath(filename)),
         "sys": sys,
+        "x86": 'x86' == platform.machine(),
+        "x86_64": 'x86_64' == platform.machine(),
+        "linux": islinux,
+        "linux32": islinux and is32bit,
+        "linux64": islinux and is64bit,
+        "armv6l": None,
+        "armv7l": None,
+        "ppc64le": None,
+        "osx": isosx,
+        "unix": islinux or isosx,
+        "win": iswin,
+        "win32": iswin and is32bit,
+        "win64": iswin and is64bit,
+        "py": int(pyversion[0] + pyversion[1]),
```

Comment:
As discussed in #74, lets remove `py`, `py2k` and `py3k` because they apply to the host Python (the conda/Python version you are using to run `conda devenv`) and not the target Python version (the Python version defined in the `environment.devenv.yml` file.

For example:

```yaml
name: myenv

dependencies:
  {% if py2k %}
  - some_lib_py2
  {% endif %} 

  {% if py3k %}
  - some_lib_py3
  {% endif %} 
  - python=3
```

This will install `my_lib_py2` if the user executing `conda devenv` using using a Python 2 `conda`, which is not the right thing to do given that the environment that will be created is for Python 3.

Reply:
Sure. I thought we should remove only `py27`, `py35`, etc.

Reply:
As to the skip ci, I used because it was a commit deleting a blank line. But since it was the last, we ended up not seeing further failed builds.